jwcrypto/__init__.py

```diff
@@ -0,0 +1,11 @@
+from jwcrypto.common import *  # noqa: F403 pylint: disable=wildcard-import
+from jwcrypto.jwa import *  # noqa: F403 pylint: disable=wildcard-import
+from jwcrypto.jwk import *  # noqa: F403 pylint: disable=wildcard-import
+from jwcrypto.jws import *  # noqa: F403 pylint: disable=wildcard-import
+from jwcrypto.jwt import *  # noqa: F403 pylint: disable=wildcard-import
+
```

Comment:
why are you exporting it all in __init__ ?
this make loading the module costly even when only one function is needed.

Reply:
The import occurs once, usually at the start of the application, and since we have the variable `__all__`, this allows us not to clutter the environment, which is recommended by the python documentation https://docs.python.org/3/tutorial/modules.html#importing-from-a-package

> Remember, there is nothing wrong with using from Package import specific_submodule! In fact, this is the recommended notation unless the importing module needs to use submodules with the same name from different packages.

thus, we hide all the internal logic by implementing the facade design pattern, which makes it easy to find the right class or function for the user and makes it easier to further support and develop library.